### PR TITLE
Cleaned up source to make golangci-lint pass

### PR DESCRIPTION
**Pull request by theonewolf**
## Description

Stylistic and whitespace changes to make `golangci-lint` pass.

## Motivation and Context

Out of the box, I couldn't get `make` to work on my system with `master`.

## How Has This Been Tested?

`make` works, and `make test` also works now.

## Checklist:

- [ ] My change requires a change to the documentation or CHANGELOG.
- [ ] I have updated the documentation/CHANGELOG accordingly.
- [x] I have created a feature (non-master) branch for my PR.

### CODE REVIEW

File: providers/azure_test.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const Post = "POST"
```
Comment (syscll) [inline providers/azure_test.go]: Instead of creating our own const, we could just use `http.MethodPost`

File: options_test.go [hunk 2]
```diff
@@ -13,12 +13,16 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const CookieSecret = "foobar"
+const ClientID = "bazquux"
+const ClientSecret = "xyzzyplugh"
```
Comment (syscll) [inline options_test.go]: These don't really need to be exported.

File: providers/azure_test.go (hunk 1 above)
Comment (theonewolf) [reply providers/azure_test.go]: Great idea.

File: options_test.go (hunk 2 above)
Comment (theonewolf) [reply options_test.go]: Agreed.

**Comment by theonewolf** [comment]
@syscll thanks for the review, I think I've addressed your earlier comments.

**Pull request merged**